starwars/src/components/Card.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+// import './Todo.css';
+function Card(props) {
+  return <div className="card"><h2 id={props.card.id}>
```

Comment:
Are you using these ids for anything? Using Id on an element is generally a bad idea because it is too specific.